demo/kserve/scripts/install/2-required-crs.sh

```diff
@@ -39,6 +39,11 @@ oc wait --for=condition=ready pod -l app=istiod -n istio-system --timeout=300s
 oc wait --for=condition=ready pod -l app=istio-ingressgateway -n istio-system --timeout=300s
 oc wait --for=condition=ready pod -l app=istio-egressgateway -n istio-system --timeout=300s
 
+# for gathering metrics
+oc apply -f custom-manifests/service-mesh/istiod-monitor.yaml 
```

Comment:
Hi @bdattoma, are these yamls required for the product to collect these metrics?

If so, shouldn't we add this to the docs as well?

Reply:
As I wrote in a previous comment, https://github.com/opendatahub-io/caikit-tgis-serving/pull/128#issuecomment-1784955112, I can't tell for sure. In our tests we don't explicitly apply these yamls but the original guide in this repo reports these steps as needed. 

Hence, I'm asking here if these are actually necessary. If not, pls close the PR and update the docs